share/scripts/inverse/run_gromacs.sh

```diff
@@ -133,14 +153,15 @@ else
   echo "${0##*/}: No walltime defined, so no time limitation given to $mdrun"
 fi
 
-#>gmx-5.1 has new handling of bonded tables
-if [[ ${mdrun_opts} != *tableb* ]]; then
+#gmx-5.1 has new handling of bonded tables - is it true only for ver.5.1 or above too???
```

Comment:
I really don't understand then why having it as you had originally could produce the error for me (gromacs ver. 5.1.2): too many input/table files provided (due to "-tableb ..." being explicitly added to the options by this piece of code). The error message is also confusing, as it suggests that I could fix it by providing the options via options.xml file, while I had already have that there, and deliberately(!) did not specify '-tableb ...' list.